src/raft.rs

```diff
@@ -285,9 +284,7 @@ impl<T: Storage> DerefMut for Raft<T> {
     }
 }
 
-trait AssertSend: Send {}
-
-impl<T: Storage + Send> AssertSend for Raft<T> {}
+impl<T: Storage + Send> Raft<T> {}
```

Comment:
This is a wrong change. `AssertSend` is used to ensure `Raft<T>` is always `Send`.

Reply:
I'm fixing this. Sorry to miss.

Reply:
Sorry for wrongly removing this constraint.